Python/generated_cases.c.h

```diff
@@ -4344,7 +4721,9 @@
             int original_opcode = 0;
             if (tstate->tracing) {
                 PyCodeObject *code = _PyFrame_GetCode(frame);
+                _PyFrame_SetStackPointer(frame, stack_pointer);
                 original_opcode = code->_co_monitoring->lines[(int)(this_instr - _PyCode_CODE(code))].original_opcode;
```

Comment:
This is another one of those "cast that looks like a call" situations. Probably need to add `int` to the list of non-escaping functions.